packages/ens-controller/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../tsconfig.packages.json",
+  "compilerOptions": {
+    "baseUrl": "./"
+  },
+  "references": [
+    { "path": "../base-controller" },
+    { "path": "../controller-utils" }
+  ],
+  "include": ["./types/**/*", "./src/**/*", "./jest.config.packages.ts"]
```

Comment:
One annoying thing I discovered was that if a dependency of a package relies on ambient types for a dependency it's importing, those ambient types will not be picked up correctly without being explicitly included. So, here `ens-controller` makes use of `controller-utils`, and `controller-utils` makes use of `eth-ens-namehash`. But `eth-ens-namehash` has no types, and so those types are filled in by `controller-utils` via an ambient definition. If those ambient types were located within `controller-utils`, then we'd have to include them here, even though this file is for `ens-controller` (TypeScript know how to find them otherwise).

To address this problem, I've decided to keep ambient types at the root and have every package always import them.